src/test/java/com/wikia/webdriver/elements/oasis/components/notifications/NotificationType.java

```diff
@@ -0,0 +1,9 @@
+package com.wikia.webdriver.elements.oasis.components.notifications;
+
+public enum NotificationType {
```

Comment:
What's the profit from keeping types in separated class instead of inside `Notifications` ?